scripts/createdownloadurls.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+with open("./sha256sums.txt") as fobj:
+    lines = fobj.readlines()
+
+
+def main():
+    filenames = []
+    for line in lines:
+        line = line.strip()
+        if not line:
+            continue
+        words = line.split()
+        if len(words) == 2:
+            filenames.append(words[1])
+        else:
+            print("Worng line: {0}".format(line))
+
+    filenames = list(set(filenames))
+    filenames.sort()
+    for name in filenames:
+        print("https://dev-bin.ops.securedrop.org/localwheels/{0}".format(name))
```

Comment:
can we provide an over-ride here (maybe env var with a default) on the base URL? 

`dev-bin.ops.securedrop.org` is for testing/development... im planning to promote wheels that are good to `bin.ops.secuerdrop.org`

Reply:
@msheiny Tell me what name you want for the envvar?